app/transport/rest/validators.go

```diff
@@ -0,0 +1,18 @@
+package rest
+
+import (
+	"regexp"
+
+	"github.com/go-devs-ua/octagon/app/entities"
+	"github.com/go-devs-ua/octagon/app/globs"
+)
+
+var idRegex = regexp.MustCompile(entities.IDMask)
+
+func validateUUID(id string) error {
+	if valid := idRegex.MatchString(id); !valid {
+		return globs.ErrInvalidID
+	}
+
+	return nil
+}
```

Comment:
Checking the UUID is not something REST-specific. Why don't you do this in `entities` package? Another validation is there already.

Reply:
fixed